examples/README.md

```diff
@@ -0,0 +1,102 @@
+# AmazonDRS Examples #
+
+This document describes the example applications provided with the [AmazonDRS library](../README.md).
+
+## Replenish example ##
+
+The example:
+- Authenticates the device on Amazon platform using the library's [login()](../README.md#logindevicemodel-deviceserial-onauthenticated-testdevice) method which provides the authentication flow described [here](../README.md#authentication)
+- Executes the following cycle: places a test order, waits for 8 sec, cancels the order, waits for 12 sec
+- Logs all responses received from the Amazon DRS
+
+**Please note that this example is not production-oriented at least due to the necessity of exposing agent's URL.**
+
+Source code: [Replenish.agent.nut](./Replenish.agent.nut)
+
+## Example Setup and Run ##
+
+### Login To Amazon ###
+
+Login to [Amazon Developer Services](https://developer.amazon.com/login.html).
+If you are not registered as a developer, create a developer's account.
+
+### Set Up Imp Device ###
+
+1. [Set up a device](https://developer.electricimp.com/gettingstarted)
+1. In the [Electric Imp's IDE](https://ide.electricimp.com) create new Product and Development Device Group.
```

Comment:
this is an old link. should update to impcentral.electricimp.com

Reply:
Fixed it